torch/csrc/jit/passes/onnx.cpp

```diff
@@ -192,24 +204,34 @@ void BlockToONNX(
 
```

Comment:
The comment is for the code above. when `is_sub_block==true`, shouldn't we avoid the above part adding input to `ctx.block`?

Reply:
Added `!is_sub_block` to skip adding input to `ctx.block`